web/src/screens/filters/List.tsx

```diff
@@ -195,17 +196,25 @@ function FilterList({ toggleCreateFilter }: any) {
     filterListState
   );
 
-  const { isLoading, data, error } = useQuery(FiltersQueryOptions(indexerFilter, sortOrder));
+  const { isLoading: isLoadingFilters, data: filtersData, error: filtersError } = useQuery(FiltersQueryOptions(indexerFilter, sortOrder));
+  const { data: indexersData, isLoading: isLoadingIndexers } = useQuery(IndexersQueryOptions());
+
+  // create a set of disabled indexer IDs for quick lookup
+  const disabledIndexerIds = useMemo(() => {
+    if (!indexersData) return new Set<number>();
+    return new Set(indexersData.filter(indexer => !indexer.enabled).map(indexer => indexer.id));
+  }, [indexersData]);
```

Comment:
Great addition!

There is however no need to fetch the indexer list again because the filters call include the indexers per filter, with the enabled property.

This can however keep the disabledIndexerIds for convenience, or just keep the current logic with `disabledIndexersInfo` in the component.